sdk/Lusid.Sdk.Tests/tutorials/Instruments/DemoEquityOption.cs

```diff
@@ -67,76 +34,47 @@ public void DemoEquityOptionCreation()
             Assert.That(roundTripEquityOption.OptionMaturityDate, Is.EqualTo(equityOption.OptionMaturityDate));
             Assert.That(roundTripEquityOption.OptionSettlementDate, Is.EqualTo(equityOption.OptionSettlementDate));
             Assert.That(roundTripEquityOption.UnderlyingIdentifier, Is.EqualTo(equityOption.UnderlyingIdentifier));
+            
+            DeleteItems(null,null,null,uniqueId);
+            
         }
 
-        [TestCase("ConstantTimeValueOfMoney")]
-        [TestCase("Discounting")]
-        [TestCase("BlackScholes")]
-        [TestCase("Bachelier")]
-        public void DemoEquityOptionValuation(string modelName)
+        [TestCase("ConstantTimeValueOfMoney", true)]
+        [TestCase("Discounting", true)]
+        [TestCase("BlackScholes", true)]
+        [TestCase("Bachelier", true)]
+        [TestCase("ConstantTimeValueOfMoney", false)]
+        [TestCase("Discounting", false)]
+        [TestCase("BlackScholes", false)]
+        [TestCase("Bachelier", false)]
+        public void DemoEquityOptionValuation(string modelName, bool inlineValuationRequest)
         {
-            var scope = $"DemoEquityOptionValuation-{modelName}";
+            // Set model and scope
+            var scope = Guid.NewGuid().ToString(); 
             var model = Enum.Parse<ModelSelection.ModelEnum>(modelName);
-
-            // CREATE and UPSERT option
+            
+            // Create instrument
             var option = (EquityOption) InstrumentExamples.CreateExampleEquityOption();
-            string uniqueId = "id-equityOption-1";
-            _testDataUtilities.UpsertOtcInstrumentToLusid(option, "some-name-for-this-equityOption", uniqueId);
-
+           
             // for pricing, we need the following market data (depending on the model)
-            _testDataUtilities.CreateAndUpsertSimpleQuote(scope, "ACME", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 135m, "USD", _demoEffectiveAt);
+            CreateAndUpsertSimpleQuote(scope, "ACME", QuoteSeriesId.InstrumentIdTypeEnum.RIC, 135m, "USD", TestDataUtilities.EffectiveAt);
```

Comment:
I think the structure of the example is obfuscating the link between the quote type and what is defined in the recipe, we need to add some commentary (or link to a KB article)

Reply:
Will see if I can make this clearer.